packages/libs/sdk/rollup.mjs

```diff
@@ -20,45 +21,66 @@ const bundle = (config) => ({
 const sharedPlugins = [
   externals(), // Exclude node_modules from bundle
   json({ compact: true }),
-  typescript({
-    tsconfig: toAbsoluteDir('tsconfig.lib.json'),
-  }),
 ];
 
 export default [
   bundle({
+    preserveModules: true,
     output: {
-      file: 'dist/packages/libs/sdk/esm/index.js',
+      dir: `${buildRootDir}/esm`,
       format: 'esm',
     },
-    plugins: [...sharedPlugins],
+    plugins: [
+      ...sharedPlugins,
+      typescript({
+        tsconfig: toAbsoluteDir('tsconfig.lib.json'),
+        outDir: `${buildRootDir}/esm`,
```

Comment:
Changed esm to output all the files instead of bundling to one so it can be tree-shaken